packages/wallet-sdk/src/relay/walletlink/type/Web3Request.ts

```diff
@@ -23,6 +23,9 @@ type _Web3Request =
         appLogoUrl: string | null;
       };
     }
+  | {
+      method: 'childRequestEthereumAccounts';
```

Comment:
another type that is used between WL extension and RN, adding it here as the new `WalletLinkEventData` tightened the types for walletlinkConnection